src/events.js

```diff
@@ -0,0 +1,51 @@
+import _ from 'lodash';
+// eslint-disable-next-line import/no-cycle
+import { addNewTask } from './class';
+// eslint-disable-next-line import/no-cycle
+import {
+  saveTaskToLocal,
+  tasks,
+  removeAll,
+  populateList,
+  listContainer,
+} from './index';
+
+export const inputTask = document.querySelector('.input-tasks');
+const clearTaskBtn = document.querySelector('.clear-btn');
+
+export const changeState = (value) => {
+  const checkBox = document.querySelectorAll('.checkbox');
+  _.forEach(checkBox, (check, i) => {
+    check.addEventListener('click', () => {
+      value[i].completed = check.checked;
+      saveTaskToLocal(value);
+      const itemDesc = Array.from(
+        document.querySelectorAll('.item-description'),
+      );
+      // eslint-disable-next-line
```

Comment:
Please ensure that all commented code are removed from your code-base, it makes your code cleaner.

Reply:
I will take care of that!